### PR TITLE
Update Polymorphic associations with many to many.md

### PR DESCRIPTION
**Pull request by afhammad**
improve wording to be more correct

### CODE REVIEW

File: guides/howtos/Polymorphic associations with many to many.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ The issue with the design above is that it breaks database references. The datab
...
 
-Luckily, the documentation for the `Ecto.Schema.belongs_to/3` macro includes a section named "Polymorphic associations" with some examples on how to design sane and performant associations. One of those approaches consists in using many join tables. Besides the "todo_lists" and "projects" tables and the "todo_items" table, we would create "todo_list_items" and "project_items" to associate todo items to todo lists and todo items to projects respectively. In terms of migrations, we are looking at the following:
+Luckily, the documentation for the `Ecto.Schema.belongs_to/3` macro includes a section named "Polymorphic associations" with some examples on how to design sane and performant associations. One of those approaches consists in using many to many join tables. Besides the "todo_lists" and "projects" tables and the "todo_items" table, we would create "todo_list_items" and "project_items" to associate todo items to todo lists and todo items to projects respectively. In terms of migrations, we are looking at the following:
```
Comment (josevalim) [inline guides/howtos/Polymorphic associations with many to many.md:98]:
```suggestion
Luckily, the documentation for the `Ecto.Schema.belongs_to/3` macro includes a section named "Polymorphic associations" with some examples on how to design sane and performant associations. One of those approaches consists in using several join tables. Besides the "todo_lists" and "projects" tables and the "todo_items" table, we would create "todo_list_items" and "project_items" to associate todo items to todo lists and todo items to projects respectively. In terms of migrations, we are looking at the following:
```

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by v0idpwn** [comment]
As a clarification, there aren't "many to many join tables" but "many join tables" as opposed to "one join table"

**Comment by afhammad** [comment]
Makes sense, thanks!

**Pull request merged**